projects/Amlogic/bootloader/config

```diff
@@ -0,0 +1,3 @@
+CONFIG_LZ4=y
```

Comment:
what is this file for?

Reply:
This is new mechanism to enable additional U-Boot options. This file is appended to the end of board defconfig.

Reply:
Adds configuration option to u-boot. This is in case we wanted to compress whole kernel image, e.g. use `Image.lzo` instead of `Image`.

Reply:
but do you actually need it right now?

Reply:
In my opinion it is needed in the same way we enable all squashfs decompression options in kernel config.

Reply:
> In my opinion it is needed in the same way we enable all squashfs decompression options in kernel config.

Sure, That's fine.